### DISCUSSION TITLE
Need Help - Error Laragon

### DISCUSSION THREAD

**Discussion by gustavopimentel** [leokhoa/laragon#508] [General]
Can anyone help me with this error. I was using Laragon with no problems until today it got this error:

Could not find procedure entry point _zend_strdup in dynamic link library

I have already removed the program and reinstalled it several times.

<img width="355" alt="Captura de tela 2023-05-22 191355" src="https://github.com/leokhoa/laragon/assets/34094677/86e6d9b3-6479-433c-992e-1f0fa56fafcf">

**Comment by chriswaterside** [comment]
I am getting the same with version Laragon 6.0.0.
 PHP 8.0 works fine but if I add PHP 8.1 and select this version of PHP I get about 10 of this type of message. Did you found a solution?

**Reply by Pen-y-Fan** [reply]
I've had the same issue, see the troubleshooting of my blog [How to update the PHP version in Laragon](https://pen-y-fan.github.io/2023/01/15/how-to-update-the-php-version-in-laragon/#troubleshooting)
